tests/test_generic_optimizer.py

```diff
@@ -702,225 +540,8 @@ def test_declearnoptimizer_09_declearn_optimizers_2_torchmodel(self):
                 loss.backward()
                 optim_w.step()
 
-    
-    def test_declearnoptimizer_10_declearn_scaffold_1_sklearnModel(self):
```

Comment:
I am ok to remove such tests, but i think we need some integration tests. current end2end tests doesnot test accurately values for scaffold